skops/card/_model_card.py

```diff
@@ -10,6 +10,7 @@
 
 from modelcards import CardData, ModelCard
 from sklearn.utils import estimator_html_repr
+from tabulate import tabulate  # type: ignore
```

Comment:
Okay, so we go with `# type: ignore` after all :D

Reply:
yes :D